Makefile

```
@@ -36,6 +36,8 @@ test:
 generate-stubs:
 	uv run stubgen -p logfire --include-docstrings --no-analysis
 	rsync -a out/logfire/ logfire-api/logfire_api/
```

```suggestion
	rsync -a out/logfire/ logfire-api/logfire_api/
	rm -rf out
```

Sometimes things in `out` from other branches stay there and have caused me confusion.